packages/jaeger-ui/src/components/Monitor/ServicesView/index.test.js

```diff
@@ -155,13 +284,20 @@ describe('<MonitorATMServicesView>', () => {
         loading: false,
         isATMActivated: true,
       },
-    });
-    expect(wrapper).toMatchSnapshot();
+      fetchServices: mockFetchServices,
+      fetchAllServiceMetrics: mockFetchAllServiceMetrics,
+      fetchAggregatedServiceMetrics: mockFetchAggregatedServiceMetrics,
+    };
+    const { container } = renderWithRouter(<MonitorATMServicesView {...snapshotProps} />);
+    expect(container.firstChild).toBeTruthy();
+    expect(screen.getAllByText('Service')[0]).toBeInTheDocument();
```

Comment:
weak test. What are we looking for, the names of services to be rendered? Then we should match on those (but rename them to apply/orange instead of s1/s2)